src/Templates/Microsoft.DotNetCore.CSharp.ProjectTemplates.Common/ClassLibrary-CSharp-NetCoreApp.vstemplate

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
+  <TemplateData>
+    <Name Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="9"/>
+    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="10"/>
+    <Icon Package="{FAE04EC1-301F-11d3-BF4B-00C04F79EFBC}" ID="4547" />
+    <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>2.0</RequiredFrameworkVersion>
+    <SortOrder>14</SortOrder>
+    <NumberOfParentCategoriesToRollUp>0</NumberOfParentCategoriesToRollUp>
```

Comment:
This needs to be 1. We want the netcore console app template to show up in the root. So it needs to roll up from the NETCore sub path.